databases/prepare_clean_psql_wildfly.sh

```diff
@@ -10,7 +10,7 @@ else
     printf "😂 Awesome, WildFly 11 is configured!\n"
```

Comment:
Not related of this PR but we should remove any specific version from this kind of file!